src/Sentry.Unity/UnityEventProcessor.cs

```diff
@@ -287,14 +287,6 @@ internal class UnityEventExceptionProcessor : ISentryEventExceptionProcessor
     {
         public void Process(Exception exception, SentryEvent sentryEvent)
         {
-            if (exception is UnityLogException ule)
```

Comment:
I guess the whole class can go now?

Reply:
`UnityLogException` is gone.
If you mean the `UnityEventExceptionProcessor`, I would rather keep it here to avoid too much churn with #683 which will put its code into this class.